relayer/app.go

```diff
@@ -386,6 +386,7 @@ func (a *App) Relay(ctx context.Context, tunnelID uint64) error {
 	tr := NewTunnelRelayer(
 		a.Log,
 		tunnel.ID,
+		tunnel.TargetChainID,
 		tunnel.TargetAddress,
```

Comment:
remove both chain and address. get them from id directly instead. prevent inconsistent data.